test/test_tensorflow.py

```diff
@@ -1423,10 +1423,6 @@ def test_horovod_alltoall_cpu(self):
         rank = hvd.rank()
         size = hvd.size()
 
-        # This test does not apply if using gloo controller
-        if hvd.gloo_enabled():
```

Comment:
There are several other alltoall tests below this that still contain this skip for gloo. Please remove from the remaining alltoall tests.

Reply:
Good catch, not sure how I missed all those.  Fixed!